### PR TITLE
Homepage content

### LINKED ISSUE #19: No content for home page
The home page needs content to introduce Bottlerocket

### PR DESCRIPTION
**Pull request by stockholmux**
**Issue number:**

Closes #19 

**Description of changes:**
Adds initial markdown content for homepage

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

**Review by mchaker — APPROVED** [review]
🚀

**Review by stmcginnis — COMMENTED** [review]
Looks great overall!

For some reason the mdlint job didn't run on this patch. There are a few things that finds that would be good to clean up:

```
content/_index.md:9 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
content/_index.md:12:392 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
content/_index.md:19:304 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
content/_index.md:23:181 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
content/_index.md:25 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
content/_index.md:32:99 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
content/_index.md:38 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
content/_index.md:41:270 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
content/_index.md:47 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
content/_index.md:52 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
content/_index.md:53 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 3]
```

Biggest issue though is images. In dark mode a lot of the text becomes unreadable. If using black text and lines, it's best not to use a transparent background and give it a solid light color.

**Comment by stockholmux** [comment]
@stmcginnis 

> For some reason the mdlint job didn't run on this patch.
and
> ... we try to follow a "one sentence per line" standard for markdown files

I haven't landed on a specific style guide for the website and if mdlint is even a good idea for the website repo specifically. Writing for a render that isn't GitHub has its own challenges and I'd like to optimize for easy contribution. But point taken about diffing and consistency. 

> Biggest issue though is images. In dark mode a lot of the text becomes unreadable.

That's fair, but maybe not in scope as it's not final. At the moment those are un-styled grayscale placeholders until we get to a colour scheme. The first goals are around content an organization rather than visual perfection. They were made in Inkscape and I'd like to natively support SVG so it would pickup whatever colours we adopt. Until we get a better structure for HTML and an automated preview, SVG is rather clunky. If you think dark mode is a requirement, feel free to put an issue and I'll put on the project board so we can track it when we get to the visual stage.

File: content/_index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Bottlerocket is a Linux-based operating system optimized for hosting containers. It’s free and open-source software, developed in the open on [GitHub](https://github.com/bottlerocket-os). Bottlerocket is installed on the machine or instance where your containers themselves are running. It is specifically designed to work with your container orchestrator (like [Kubernetes](https://kubernetes.io/)) to automate the lifecycle of the containers running in your cluster. Bottlerocket runs in the cloud or in your datacenter.
```
[the remark this answers is not in the document]
Comment (stockholmux) [reply content/_index.md:1]: See https://github.com/bottlerocket-os/project-website/pull/20#issuecomment-1408951315

File: content/_index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Bottlerocket has three primary benefits goals: Minimal · Safe Updates · Security Focused.
```
[the remark this answers is not in the document]
Comment (stockholmux) [reply content/_index.md:3]: That's a good catch.

**Comment by stmcginnis** [comment]
> Writing for a render that isn't GitHub

That is not what mdlinting is about.

We target GitHub and Hugo, and right now they may render fine. But each of them use libraries to perform that rendering, which could be updated at any point. This has, and could, cause slight changing in how the rendered output is displayed.

So other than writing "correct" markdown being.. the more correct thing to do, having mdlint verify things helps ensure that some dependency update somewhere in the change doesn't suddenly break our site.

**Comment by stockholmux** [comment]
@stmcginnis I'm not sure I agree. 

mdlint demands a lot that amounts to little: Example: `no-trailing-spaces` has nothing to do with rendering nor writing correct markdown (nothing mentioned about it in the CommonMark spec). If the project needs to be consistent, that's fine and a different conversation, but we can't kid ourselves that it's something more than that.

As far as a renderer changing how it interprets the markdown, I have few worries about that. If changes in how, say, a paragraph is rendered, that seems like an acceptable risk - especially if it lowers the effort and hassle needed for those contributing. On top of that, in a statically rendered site that where the dependencies are controllable - it seems like a very low concern.

**Pull request merged**